packages/vue-components/src/SiteNavButton.vue

```diff
@@ -51,15 +57,59 @@ export default {
 </script>
 
 <style scoped>
+
     .toggle-site-nav-button {
         cursor: pointer;
+        height: 53px;
         left: 0;
         padding: 15px;
         position: absolute;
+        transform: all rotate(0deg);
+        transition: all 0.25s ease-in-out;
+        width: 52px;
+    }
+
+    .toggle-site-nav-button span {
+        background-color: #000;
+        display: block;
+        height: 3px;
+        left: 15px;
+        position: absolute;
+        top: 50%;
+        transform: all rotate(0deg);
+        transition: all 0.25s ease-in-out;
+        width: 21px;
+    }
+
+    .toggle-site-nav-button > span:nth-child(1) {
+        transform: translateY(-6px);
+    }
+
+    .toggle-site-nav-button > span:nth-child(2) {
+        transform-origin: 100% 50%;
     }
 
-    .toggle-site-nav-button::before {
-        content: "\e236";
-        font-size: 20px;
+    .toggle-site-nav-button > span:nth-child(3) {
+        transform: translateY(6px);
     }
+
+    .nav-menu-close-icon .toggle-site-nav-button > span:nth-child(1) {
+        -o-transform: rotate(135deg);
```

Comment:
Thanks for including these. 🙂 We have many other instances of missing vendor prefixes however -- I think MarkBind generally hasn't been too particular on browser compatibility so far.

I wonder if it'd be better to remove these to lessen the noise and for consistency with this pattern. (am using the `unprefixed` metric in here https://caniuse.com/?search=transform to gauge vendor prefix support.)

Reply:
Updated to make it consistent with the pattern😊